heareval/tasks/speech_commands.py

```diff
@@ -104,13 +104,15 @@ def run(self):
         for file_obj in train_path.iterdir():
             if file_obj.is_dir() and file_obj.name != BACKGROUND_NOISE:
                 linked_folder = Path(os.path.join(self.workdir, file_obj.name))
-                linked_folder.unlink(missing_ok=True)
+                if linked_folder.exists():
+                    linked_folder.unlink(missing_ok)
```

Comment:
`missing_ok` is not defined here. Did you intend to remove it?

Reply:
oops yes I should have removed it